app/star-rating.sass

```diff
@@ -9,4 +9,32 @@ h1
    margin: 0
 
 #app
-   font-size: 3em
\ No newline at end of file
+   font-size: 3em
+
+.star-rating
+    display: inline-block
+    unicode-bidi: bidi-override
```

Comment:
Can you help me understand what you are doing with the `unicode-bidi` property? Was playing around in browser and enabling/disabling didn't seem to do anything.

Reply:
I am using the unicode-bidi property in conjunction with the direction property to reverse the order of the star characters.  https://developer.mozilla.org/en-US/docs/Web/CSS/unicode-bidi

In order to have the previous stars change on hover or click, I chose to reverse the order of the text display. I was unable to find a CSS selector which chooses all previous siblings, but the selector "~" selects all successor siblings. The hover and click functions are achieved through selecting all succeeding siblings on hover combined with the reversed order of the characters.

Disabling the unicode-bidi should result in hover selecting all stars to the right, and not left, as intended. If there is different behavior on hover/click with the unicode-bidi disabled, I would be interested to look at it to try and repro the behavior.